app/auth/forms.py

```diff
@@ -19,14 +19,20 @@ class RegistrationForm(FlaskForm):
     submit = SubmitField('Register')
 
     def validate_username(self, username):
-        user = User.query.filter_by(username=username.data).first()
+        user = User.query.filter(User.username.like(username.data)).first()
```

Comment:
Using 'like' here feels a little non-specific for the validation we're trying to enforce.  I think readability/clarity would be increased if we just added something like 
```
username_lower = username.lower()
user = User.query.filter_by(username_lower=username.data).first()
```